script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -101,7 +101,7 @@ The doctor clattered across the floor and vanished behind a curtain.
 <Page28>
 "...It seems there is some brain damage after all.
 Contact Doctor Ashiya in the neurosurgery department. I also suspect some damage to the optic nerves. Send him in for an eye examination this afternoon."
-The doctor whispered to the nurse in a cold voice, as if he was an entirely different person.
+The doctor whispered to the nurse in a cold voice, as if he were an entirely different person.
```

Comment:
> The doctor whispered to the nurse in a cold voice, like an entirely different person.

Reply:
Hm, this one doesn't seem as good to me. Either as-is or "a cold voice, like **that of** an entirely different person"

Reply:
like that of is fine